### PR TITLE
Run shared integration tests on Travis

### PR DESCRIPTION
**Pull request by dansimau**
This PR adds an npm script `npm run test-shared-integration-tests`. This script runs the shared integration tests from the ringpop-common repository.

It also adds the running of these tests to Travis CI.

### CODE REVIEW

File: test/run-shared-integration-tests [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+# Clones or updates the ringpop-common repository.
+#
+fetch-ringpop-common() {
+    if [ ! -e "$ringpop_common_dir" ]; then
+        run git clone --depth=1 https://github.com/uber/ringpop-common.git "$ringpop_common_dir"
+    fi
+
+    run cd "$ringpop_common_dir"
+    run git pull
+    run cd - >/dev/null
```
Comment (mennopruijssers) [inline test/run-shared-integration-tests]: Could be in an else right?

**Comment by severb** [comment]
What is this wizardry? LGTM 🎩

**Comment by dansimau** [comment]
@severb glad you didn't ask where the tests were...

**Comment by thanodnl** [comment]
For ringpop-go we are about to land a change that lets us pick a feature branch for integration tests during development: https://github.com/uber/ringpop-go/blob/feature/reaping-faulty-nodes/test/run-integration-tests#L9

would it be possible to port that to this one as well? It has proven really useful in adding new integration tests.

**Comment by mennopruijssers** [comment]
LGTM. +1 for adding (e.g. an env-variable) to specify the branch of ringpop-common to checkout.

**Comment by dansimau** [comment]
+1 too. I filed something internally for us to follow up later.

**Comment by thanodnl** [comment]
LGTM, thanks for porting the branch selection.

**Pull request merged**